apps/openmw/mwworld/store.cpp

```diff
@@ -282,7 +282,7 @@ namespace MWWorld
             typename std::vector<T *>::iterator end = sharedIter + mStatic.size();
 
             while (sharedIter != mShared.end() && sharedIter != end) {
-                if((*sharedIter)->mId == idLower) {
+                if ((*sharedIter)->mId == idLower) {
```

Comment:
Please avoid to touch lines if you really to do not modify it.

Reply:
Hmm, weird. I blame this one on my IDE.